app/assets/javascripts/account/AccountController.js.coffee

```diff
@@ -272,6 +273,20 @@ AccountController = (
     else
       false
 
+  $scope.validatePasswordConfirmationMatch = (value) ->
```

Comment:
@software-project are you saying that we can't check for password match if the first password is invalid because the value doesn't actually get set?